opt/scopeserver/settings.py

```diff
@@ -0,0 +1,14 @@
+# settings.py
+import os
+from dotenv import load_dotenv
+from pathlib import Path
+
+env_path = Path("..") / ".env"
+load_dotenv(dotenv_path=env_path)
```

Comment:
This isn't necessary since it can be run with `poetry run dotenv -f $ENV_FILE run ...`. Also, [Pydantic has `.env` support](https://pydantic-docs.helpmanual.io/usage/settings/#dotenv-env-support)